slack/message.w

```diff
@@ -0,0 +1,49 @@
+// TODO: right now there is no way to define how an enum
```

Comment:
Do we have a tracking issue?

Reply:
I did not find one, opened: https://github.com/winglang/wing/issues/6470